src/linear_algebra/dfinsupp.lean

```diff
@@ -4,7 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Kenny Lau
 -/
 import data.dfinsupp
```

Comment:
Let's keep the imports ordered (sorry for two suggestions, GitHub won't let you make suggestions involving deleted lines.)
```suggestion
import data.finsupp.to_dfinsupp
```